modin/pandas/test/test_io.py

```diff
@@ -278,6 +275,8 @@ def test_read_csv_col_handling(
             pytest.xfail(
                 reason="The reason of tests fail in `cloud` mode is unknown for now - issue #2340"
             )
+        if names is lib.no_default:
```

Comment:
shouldn't this be checked for omnisci only or something?

Reply:
Not sure, tests with non-default `names` still fail with pandas-on-Ray/Dask backends.

`names` `None` parameter was changed to `lib.no_default` to avoid exceptions `ValueError: Specified named and prefix; you can only specify one.` (that were raised both by pandas and Modin that leads to tests passing by just checking exception types) and it's occurred that only remained failed tests are tests with `names=lib.no_default`, so they were skipped.
Honestly, this change not directly related to PR issue, but enabled tests are checking PR changes, so i prefer leave these changes in this PR.